test_helpers/assert-clone-of.js

```diff
@@ -0,0 +1,12 @@
+import { cloneElement } from 'preact';
```

Comment:
I put this file in a new `test_helpers` directory, because it's not a test itself, but it's imported by some of the tests. If you'd prefer it live within the `test` directory, or a subdirectory inside `test` (e.g. `test/_helpers`), I'd be happy to move it there. We'll just need to modify [`files: ['test/**/*.js']` in the karma config](https://github.com/developit/preact-router/blob/13e44beb6c14f7905f746eeb61535eff03bff572/karma.conf.js#L7) to exclude this if we move it inside `test`.